src/models/Transaction/index.ts

```diff
@@ -36,3 +36,9 @@ export interface Transaction {
   maxCycles: number | null
   createTimestamp?: number
 }
+
+export interface BtxTx {
+  txId: string
+  transactionHash: string
+  ckbTransactionHash: string
```

Comment:
`txId` is the index of btc transaction and `ckbTransactionHash` is the index of ckb transaction, but what is `transactionHash` for

Reply:
Bitcoin transaction hash, example:

```bash
curl --location --request POST 'https://go.getblock.io/979e35393afe4ad6b43283d6c996fc28' \
--header 'User-Agent: Apifox/1.0.0 (https://apifox.com)' \
--header 'Content-Type: application/json' \
--header 'Accept: */*' \
--header 'Host: go.getblock.io' \
--header 'Connection: keep-alive' \
--data-raw '{
    "method": "getrawtransaction",
    "params": [
        "15dede3b31ed87bb6b1d668222127a7b308c1beb6fe99bf4a3f076bcae8e93fe",
        2
    ]
}'
```

```json
...
"txid": "15dede3b31ed87bb6b1d668222127a7b308c1beb6fe99bf4a3f076bcae8e93fe",
"hash": "75f0e862cea511515b34f140a4a328c2bd32d09d49f6ca0dc99f9da925e5e645",
"version": 1,
"size": 392,
"vsize": 365,
"weight": 1460
...
```

Reply:
Got it, `txId` and `transactionHash` are fields of btc transaction.
It may bring from [bip-0141](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki)
Here is the talk about `txId` and `hash`